microceph/ceph/config.go

```diff
@@ -45,8 +46,28 @@ func (c ConfigTable) Keys() (keys []string) {
 // so that each request for the map guarantees consistent definition.
 func GetConstConfigTable() ConfigTable {
 	return ConfigTable{
+		// Cluster config keys
 		"cluster_network":             {"global", []string{"osd"}},
 		"osd_pool_default_crush_rule": {"global", []string{}},
+		// RGW config keys
+		"rgw_keystone_url":                            {"global", []string{"rgw"}},
+		"rgw_keystone_admin_token":                    {"global", []string{"rgw"}},
+		"rgw_keystone_admin_token_path":               {"global", []string{"rgw"}},
+		"rgw_keystone_admin_user":                     {"global", []string{"rgw"}},
+		"rgw_keystone_admin_password":                 {"global", []string{"rgw"}},
+		"rgw_keystone_admin_password_path":            {"global", []string{"rgw"}},
+		"rgw_keystone_admin_tenant":                   {"global", []string{"rgw"}},
+		"rgw_keystone_admin_project":                  {"global", []string{"rgw"}},
+		"rgw_keystone_admin_domain":                   {"global", []string{"rgw"}},
+		"rgw_keystone_service_token_enabled":          {"global", []string{"rgw"}},
+		"rgw_keystone_service_token_accepted_roles":   {"global", []string{"rgw"}},
+		"rgw_keystone_expired_token_cache_expiration": {"global", []string{"rgw"}},
+		"rgw_keystone_api_version":                    {"global", []string{"rgw"}},
+		"rgw_keystone_accepted_roles":                 {"global", []string{"rgw"}},
+		"rgw_keystone_accepted_admin_roles":           {"global", []string{"rgw"}},
+		"rgw_keystone_token_cache_size":               {"global", []string{"rgw"}},
+		"rgw_keystone_verify_ssl":                     {"global", []string{"rgw"}},
+		"rgw_keystone_implicit_tenants":               {"global", []string{"rgw"}},
```

Comment:
Can you add config rgw `swift account in url`. This is used in conjunction with `rgw_keystone_implicit_tenants`. [1]
[1] https://opendev.org/openstack/charm-ceph-radosgw/src/branch/master/templates/ceph.conf#L80